app/vue/src/client/preview/story_store.js

```diff
@@ -0,0 +1,89 @@
+/* eslint no-underscore-dangle: 0 */
+
+let count = 0;
+
+function getId() {
+  count += 1;
+  return count;
+}
+
+export default class StoryStore {
```

Comment:
We could reuse almost everything except the rendered. We should have @storybook/app with generic stuff and react / react-native /... will just change the renderer. 

But first we need to validate that vue can have the exact same api :)